view.go

```diff
@@ -225,6 +236,27 @@ func (v *View) SetCursor(x, y int) error {
 	return nil
 }
 
+// SetViewLineUp sets the cursor position of the view at the one line up
+func (v *View) SetViewLineUp() {
+	if v.cy > 0 {
+		v.cy -= 1
+	} else if v.oy > 0 {
+		v.oy -= 1
+	}
+}
+
+// SetViewLineDown sets the cursor position of the view at the one line down
+func (v *View) SetViewLineDown() {
+	_, maxY := v.Size()
+	if v.cy+v.oy < v.ViewLinesHeight()-1 {
```

Comment:
Can you change this code to be like this, so we don't have nested ifs:
```suggestion
	if v.cy+v.oy >= v.ViewLinesHeight()-1 {
		return
	}
```